amida-tech/indaba-client
# [INBA-948] Changes that allow for password reset to be accurate.

#### What does this PR do?
Fixes a bug with Indaba auth reset password.

#### Related JIRA tickets:
INBA-948

#### How should this be manually tested?
First, go into the `.env` for the `develop` branch of your `amida-auth-microservices`. Add the following variables:
`AUTH_SERVICE_MAILER_EMAIL_ID=indaba-mailer-dev@amida.com`
`AUTH_SERVICE_MAILER_PASSWORD=''`
`AUTH_SERVICE_MAILER_FROM_EMAIL_ADDRESS=indaba-mailer-dev@amida.com`
`AUTH_SERVICE_MAILER_SERVICE_PROVIDER=Gmail`

James will message you the password.

Then start up your Indaba and all related services. If you don't already, create a user whose email address you have access to. Log out of admin and at the login page, enter the email address of your newly created user. Instead of logging in, click "Forgot Password" and say yes to the modal that appears. Check that you receive an email with a link to reset your password. Complete the reset password process.

#### Background/Context

#### Screenshots (if appropriate):
